File: Array/3190. Find Minimum Operations to Make All Elements Divisible by Three/minimum_operations.py
from typing import List
class Solution:
    def minimumOperations(self, nums: List[int]) -> int:
        result=0
        for num in nums:
            if num%3!=0:
                result+=1
        return result
    
if __name__ == "__main__":
    sol = Solution()
    nums1 = [1, 2, 3, 4]
    nums2 = [3, 6, 9]

    print(f"Minimum operations for nums1: {sol.minimumOperations(nums1)}")
    print(f"Minimum operations for nums2: {sol.minimumOperations(nums2)}")

File: Array/3190. Find Minimum Operations to Make All Elements Divisible by Three/minimum_operations.java
class Solution {
    public int minimumOperations(int[] nums) {
        int result=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]%3!=0){
                result++;
            }
        }
        return result;
    }
    public static void main(String[] args) {
        Solution sol = new Solution();
        int[] nums1 = {1, 2, 3, 4};
        int[] nums2 = {3, 6, 9};

        System.out.println("Minimum operations for nums1: " + sol.minimumOperations(nums1));
        System.out.println("Minimum operations for nums2: " + sol.minimumOperations(nums2));
    }
}